crates/miden-objects/src/account/code/mod.rs

```diff
@@ -524,9 +524,8 @@ mod tests {
 
     #[test]
     fn test_account_code_no_auth_component() {
-        let component = AccountComponent::compile(CODE, Assembler::default(), vec![])
-            .unwrap()
-            .with_supports_all_types();
+        let library = Assembler::default().assemble_library([CODE]).unwrap();
+        let component = AccountComponent::new(library, vec![]).unwrap().with_supports_all_types();
```

Comment:
I removed `AccountComponent::compile`, but because these tests live in `miden-objects` we can't get around using `Assembler`